cookbooks/bcpc/recipes/powerdns.rb

```diff
@@ -28,13 +28,12 @@
 results = get_nodes_for("powerdns").map!{ |x| x['fqdn'] }.join(",")
 nodes = results == "" ? node['fqdn'] : results
 
+#
+# For some reason, we are compelled to specify a provider.
+# This will probably break if we ever move to chef-vault cookbook 2.x
```

Comment:
chef-vault cookbook 2.x rewrote the provider from scratch using Chef 12.5+ "Custom Resources," so if it happens to have the same provider name as the cookbook 1.x incarnation, that is almost totally accidental.